code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/AEPMessage.java

```diff
@@ -116,7 +127,7 @@ class AEPMessage implements FullscreenMessage {
 
     @Override
     @Nullable public WebView getWebView() {
-        return this.webView;
+        return webView != null ? webView : createWebView();
```

Comment:
The getter should only return `webView` and not create a new instance.

Reply:
my bad, thought i had cleaned this